### PR TITLE
[Dynamic typebindings] Remove Unreal SDK and C++ codegen

### PR DESCRIPTION
**Pull request by improbable-valentyn**

### CODE REVIEW

**Comment by Vatyx** [comment]
pogchamp

**Comment by Vatyx** [comment]
I think BuildWorker has to change? Don't we call Codegen in there or something?

File: Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbarSettings.cpp [hunk 1]
```diff
@@ -21,12 +20,10 @@ FString USpatialGDKEditorToolbarSettings::ToString()
...
 	Args.Add(SpatialOSSnapshotPath.Path);
 	Args.Add(SpatialOSSnapshotFile);
-	Args.Add(InteropCodegenOutputFolder.Path);
 	Args.Add(GeneratedSchemaOutputFolder.Path);
 
 	return FString::Format(TEXT("ProjectRootFolder={0}, SpatialOSLaunchArgument={1}, "
 								"bStopSpatialOnExit={2}, SpatialOSSnapshotPath={3}, "
-								"SpatialOSSnapshotFile={4}, InteropCodegenOutputFolder={5}"
-								"GeneratedSchemaOutputFolder={6}"),
+								"SpatialOSSnapshotFile={4}, GeneratedSchemaOutputFolder={6}"),
```
Comment (Vatyx) [inline Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbarSettings.cpp]: Probably change GeneratedSchemaOutputFolder to 5?

**Comment by improbable-valentyn** [comment]
`BuildWorker` just calls `Build.exe` and it's the `Build.cs` that used to call Codegen (no longer does)

**Comment by Vatyx** [comment]
There is an issue Josh, I and Peter ran into where standard library schema was missing. I think your change in BuildGDK should fix that.

**Comment by Vatyx** [comment]
Could we eliminated Build.cs altogether and make that file batch?

**Comment by improbable-valentyn** [comment]
> Could we eliminated Build.cs altogether and make that file batch?

We could. Should probably happen in a separate PR.

File: Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp [hunk 2]
```diff
@@ -83,13 +79,13 @@ bool CheckClassNameListValidity(const TArray<UClass*>& Classes)
 }
 }// ::
 
-void GenerateInteropFromClasses(const TArray<UClass*>& Classes, const FString& CombinedSchemaPath, const FString& CombinedForwardingCodePath)
+void GenerateInteropFromClasses(const TArray<UClass*>& Classes, const FString& CombinedSchemaPath)
```
Comment (joshuahuburn) [inline Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp]: Probably want to rename this function now.

File: Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbarSettings.h [hunk 3]
```diff
@@ -47,10 +47,6 @@ class USpatialGDKEditorToolbarSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Configuration", meta = (ConfigRestartRequired = false, DisplayName = "Snapshot file name"))
 	FString SpatialOSSnapshotFile;
 
-	/** Interop codegen output path */
-	UPROPERTY(EditAnywhere, config, Category = "Interop codegen", meta = (ConfigRestartRequired = false, DisplayName = "Output path for the interop codegeneration"))
-	FDirectoryPath InteropCodegenOutputFolder;
-
 	/** Generated schema output path */
 	UPROPERTY(EditAnywhere, config, Category = "Interop codegen", meta = (ConfigRestartRequired = false, DisplayName = "Output path for the generated schemas"))
```
Comment (joshuahuburn) [inline Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbarSettings.h]: `"Interop codegen"` I wonder if this is still a relevant name now.

**Comment by improbable-valentyn** [comment]
I imagine we'll need to make another pass to fix up all the names in code as well as docs.

**Pull request merged**